#service for applying the wrenches to gazebo model

last_letter_2/Model_wrenches model_wrenches
---
bool success